doc/nrf/known_issues.rst

```diff
@@ -1048,6 +1109,43 @@ DRGN-13921: Directed advertising issues using RPA in TargetA
 DRGN-10367: Advertiser times out earlier than expected
   If an extended advertiser is configured with limited duration, it will time out after the first primary channel packet in the last advertising event.
 
+.. rst-class:: v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-11222: Master role can cause disconnects if there are scheduling conflicts while doing a control procedure with an instant.
+  This bug will be triggered if all the following conditions are true:
+
+  * When a procedure instant is active (control PDUs have been exchanged but the instant is not reached yet)
+  * The link manager of master gets blocked
+  * The link manager detects that more than one of its events were blocked
```

Comment:
I would suggest out to leave out the details here as it is not possible for the reader of the document to understand these conditions.
```suggestion
DRGN-11222: Master connections can in rare cases disconnect prematurely if there are scheduling conflicts while doing a control procedure with an instant.
```